charts/portworx/templates/portworx-operator.yaml

```diff
@@ -0,0 +1,99 @@
+{{- if .Values.deployOperator }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: portworx-operator
+  namespace: kube-system
```

Comment:
A very dumb question. Does it need to be in kube-system, can user install the operator and the resources in other namspaces?

Reply:
technically it could be in other namespace, but since daemonset only uses kube-system, we use it as default